cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -163,6 +163,8 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, k8sVersion *versio
 		"requestheader-allowed-names":        "front-proxy-client",
 		"proxy-client-cert-file":             filepath.Join(cfg.CertificatesDir, kubeadmconstants.FrontProxyClientCertName),
 		"proxy-client-key-file":              filepath.Join(cfg.CertificatesDir, kubeadmconstants.FrontProxyClientKeyName),
+		// add "admissionregistration.k8s.io/v1alpha1" to the --runtime-config flag due to Initializers is enabled
```

Comment:
nit: replace `due to Initializers is enabled` with `because Initializers are enabled`